io.github.cnruby.main_139.json

```diff
@@ -0,0 +1,35 @@
+{
+  "app-id": "io.github.cnruby.main_139",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "main_139",
+  "modules": [
+      {
+          "name": "main_139",
+          "only-arches": [
+            "x86_64"
+          ],
+          "cmake": true,
+          "buildsystem": "cmake-ninja",
+          "builddir": true,
+          "_comment": "-DCMAKE_INSTALL_LIBDIR=lib,",
+          "config-opts": [
+            "-DALLOW_IN_SOURCE_BUILD=ON",
+            "-DCMAKE_INSTALL_PREFIX=/app",
+            "-DCMAKE_BUILD_TYPE=Release"
+          ],
+          "build-commands": [
```

Comment:
you only use this section if you're using the "simple" buildsystem, otherwise everything else happens "magically".

Reply:
I use the "cmake-ninja" buildsystem

Reply:
I changed the follow code. Why are there still other arches?
```
          "only-arches": [
            "x86_64"
          ],
```